grammar.js

```diff
@@ -373,6 +378,39 @@ module.exports = grammar({
       seq(repeat($._annotation), '[', ']')
     )),
 
+    switch_construct: $ => seq(
+      'switch',
+      field('condition', $.parenthesized_expression),
+      field('body', $.switch_block)
+    ),
+
+    switch_block: $ => seq(
+      '{',
+      choice(repeat($.switch_line), repeat($.arrow_switch_line)),
+      '}'
+    ),
+
+    switch_line: $ => choice(
```

Comment:
I don't think we should have this `switch_line` node be visible in the tree. What do you think about simply eliminating it, since it only is used once?

Reply:
I ended up renaming it to match Java's spec https://docs.oracle.com/javase/specs/jls/se16/html/jls-14.html#jls-SwitchBlock